wp/plugin/wp.ml

```diff
@@ -18,149 +18,158 @@ module Typ = Extension.Type
 
 let postcond = Cmd.parameter Typ.string "postcond"
-    ~doc:"A custom postcondition in SMT-LIB format to be used during the \
-          analysis. If no postcondition is specified, a trivial \
-          postcondition of `true' will be used."
+    ~doc:{|Replaces the default postcondition with the user-specified one,
+           using the smt-lib2 format. Similar to `--precond', one may create
+           comparative postconditions on variables by appending `_orig' and
+           `_mod' to variable names. If no postcondition is specified, a
+           trivial postcondition of `true' will be used.|}
 
 let trip_asserts = Cmd.flag "trip-asserts"
-    ~doc:"If set, WP will look for inputs to the subroutine that would cause \
-          an __assert_fail or __VERIFIER_error to be reached."
+    ~doc:{|Looks for inputs to the subroutine that would cause an
+           `__assert_fail' or `__VERIFIER_error' to be reached.|}
 
 let check_null_derefs = Cmd.flag "check-null-derefs"
-    ~doc:"If set, the WP analysis will check for inputs that would result in \
-          dereferencing a NULL value. In the case of a comparative analysis, \
-          WP will check that the modified binary has no additional null \
-          dereferences in comparison with the original binary."
+    ~doc:{|Checks for inputs that would result in dereferencing a null value
+           during a memory read or write. In the case of a comparative
+           analysis, checks that the modified binary has no additional null
+           dereferences in comparison with the original binary.|}
 
 let compare_func_calls = Cmd.flag "compare-func-calls"
-    ~doc:"This flag is used for a comparative analysis. If set, WP will \
-          check that function calls should not occur in the modified binary \
-          if they have not occurred in the original binary."
+    ~doc:{|This flag is only used in a comparative analysis. Checks that
+           function calls do not occur in the modified binary if they have not
+           occurred in the original binary.|}
 
 let compare_post_reg_values = Cmd.parameter Typ.(list string) "compare-post-reg-values"
-    ~doc:"This flag is used for a comparatve analysis. If set, WP will \
-          compare the values stored in the specified registers at the end of \
-          the analyzed function's execution. For example, `RAX,RDI' compares \
-          the values of RAX and RDI at the end of execution. If unsure about \
-          which registers to compare, x86_64 architectures place their \
-          output in RAX, and ARM architectures place their output in R0."
+    ~doc:{|This flag is only used in a comparatve analysis. Compares the values
+           stored in the specified registers at the end of the function's
+           execution. For example, `RAX,RDI' compares the values of RAX and RDI
+           at the end of execution. If unsure about which registers to compare,
+           x86_64 architectures place their output in RAX, and ARM
```

Comment:
Maybe say "If unsure about which registers to compare, check the architecture's ABI. x86_64 architectures place their output in RAX, ..."